jsHelper/spicetifyWrapper.js

```diff
@@ -980,101 +980,141 @@ Spicetify.ContextMenu = (function () {
-                }
-            }
-        } else if (parent.props.children) {
-            return findChild(parent.props.children, key, value);
-        }
-        return null;
-    }
-
-    function conditionalAppend(baseClassname, conditionalClassname, location) {
-        if (Spicetify.Platform?.History?.location && Spicetify.Platform.History.location.pathname.startsWith(location)) {
-            baseClassname += " " + conditionalClassname;
-        }
-
-        return baseClassname;
-    }
-
-    const React = Spicetify.React;
-    const reactObjs = [];
-    for (const app of list) {
-        let manifest;
-        try {
-            var request = new XMLHttpRequest();
-            request.open('GET', `spicetify-routes-${app}.json`, false);
-            request.send(null);
-            manifest = JSON.parse(request.responseText);
-        } catch {
-            manifest = {};
-        }
-
-        let appProper = manifest.name;
-        if (typeof appProper === "object") {
-            appProper = appProper[Spicetify.Locale.getLocale()] || appProper["en"];
-        }
-        if (!appProper) {
-            appProper = (app[0].toUpperCase() + app.slice(1));
-        }
-        const icon = manifest.icon || "";
-        const activeIcon = manifest["active-icon"] || icon;
-
-        const appLink = "/" + app;
-        const link = findChild(Spicetify._sidebarItemToClone, "className", "main-navBar-navBarLink");
-        const obj = React.cloneElement(
-            Spicetify._sidebarItemToClone,
-            null,
-            React.cloneElement(
-                link,
-                {
-                    to: appLink,
-                    isActive: (e, {pathname: t})=> t.startsWith(appLink),
-                    className: conditionalAppend("link-subtle main-navBar-navBarLink", "main-navBar-navBarLinkActive", appLink),
-                },
-                React.createElement(
-                    "div",
-                    {
-                        className: "icon collection-icon",
-                        dangerouslySetInnerHTML: {
-                            __html: icon,
-                        }
-                    },
-                ),
-                React.createElement(
-                    "div",
-                    {
-                        className: "icon collection-active-icon",
-                        dangerouslySetInnerHTML: {
-                            __html: activeIcon,
-                        }
-                    },
-                ),
-                React.createElement(
-                    "span",
-                    {
-                        className: "ellipsis-one-line main-type-mestoBold"
-                    },
-                    appProper,
-                ),
-            )
-        )
-        reactObjs.push(obj);
-    }
-    return reactObjs;
-}
+Spicetify._cloneSidebarItem = function (list, sidebarIsCollapsed) {
+	function findChild(parent, key, value) {
+		if (!parent.props) return null;
```

Comment:
If you use brackets below for one line ifs, do it there too

Reply:
This was unchanged from the original diff so I didn't make any changes.
If you want to refactor it also then I'll do it later.

Reply:
Well, I prefer no brackets for one line ifs, but it would mean refactoring all javascript files